<ui version="4.0" >
 <author>Markus Knapp</author>
 <class>Gui</class>
 <widget class="QWidget" name="Gui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1441</width>
    <height>851</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>direcs remote</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../direcs-remote.qrc" >
    <normaloff>:/images/images/underFootOne.png</normaloff>:/images/images/underFootOne.png</iconset>
  </property>
  <property name="layoutDirection" >
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QGroupBox" name="groupBoxControl" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title" >
    <string>Control</string>
   </property>
   <widget class="QPushButton" name="btnBackward" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>280</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="icon" >
     <iconset resource="../direcs-remote.qrc" >
      <normaloff>:/images/images/down.png</normaloff>:/images/images/down.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>82</width>
      <height>82</height>
     </size>
    </property>
    <property name="shortcut" >
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLeft" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>150</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="icon" >
     <iconset resource="../direcs-remote.qrc" >
      <normaloff>:/images/images/back.png</normaloff>:/images/images/back.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>82</width>
      <height>82</height>
     </size>
    </property>
    <property name="shortcut" >
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRight" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>150</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="icon" >
     <iconset resource="../direcs-remote.qrc" >
      <normaloff>:/images/images/forward.png</normaloff>:/images/images/forward.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>62</width>
      <height>62</height>
     </size>
    </property>
    <property name="shortcut" >
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartStop" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>150</y>
      <width>111</width>
      <height>110</height>
     </rect>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip" >
     <string>drive</string>
    </property>
    <property name="statusTip" >
     <string>drive</string>
    </property>
    <property name="whatsThis" >
     <string>drive</string>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="icon" >
     <iconset resource="../direcs-remote.qrc" >
      <normaloff>:/images/images/underFootOne.png</normaloff>:/images/images/underFootOne.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>82</width>
      <height>82</height>
     </size>
    </property>
    <property name="shortcut" >
     <string>5</string>
    </property>
    <property name="autoDefault" >
     <bool>true</bool>
    </property>
    <property name="default" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnForward" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>20</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="icon" >
     <iconset resource="../direcs-remote.qrc" >
      <normaloff>:/images/images/up.png</normaloff>:/images/images/up.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>82</width>
      <height>82</height>
     </size>
    </property>
    <property name="shortcut" >
     <string>8</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxSpeed1" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>255</number>
    </property>
    <property name="value" >
     <number>23</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxSpeed2" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>20</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>255</number>
    </property>
    <property name="value" >
     <number>23</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShutdown" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>280</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>shutdown</string>
    </property>
    <property name="statusTip" >
     <string>shutdown</string>
    </property>
    <property name="whatsThis" >
     <string>shutdown</string>
    </property>
    <property name="icon" >
     <iconset resource="../direcs-remote.qrc" >
      <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>82</width>
      <height>82</height>
     </size>
    </property>
    <property name="shortcut" >
     <string>2</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxNetwork" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>430</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Network</string>
   </property>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPortValue" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>45454</string>
    </property>
    <property name="maxLength" >
     <number>6</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTest" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;Test</string>
    </property>
    <property name="shortcut" >
     <string>Alt+T</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxLog" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>550</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title" >
    <string>Log</string>
   </property>
   <widget class="QTextEdit" name="textEdit1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;Clear</string>
    </property>
    <property name="shortcut" >
     <string>Alt+C</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnExit" >
   <property name="geometry" >
    <rect>
     <x>1060</x>
     <y>730</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="icon" >
    <iconset resource="../direcs-remote.qrc" >
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="shortcut" >
    <string>Alt+X</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLink" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>730</y>
     <width>83</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text" >
    <string>www.direcs.de</string>
   </property>
   <property name="textFormat" >
    <enum>Qt::RichText</enum>
   </property>
   <property name="openExternalLinks" >
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags" >
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxWeb" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>10</y>
     <width>701</width>
     <height>711</height>
    </rect>
   </property>
   <property name="title" >
    <string>Web</string>
   </property>
   <widget class="QWidget" native="1" name="widgetWeb" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>681</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGo" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>680</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;Go!</string>
    </property>
    <property name="shortcut" >
     <string>Alt+T</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>630</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditWebAddress" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>650</y>
      <width>681</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSensors" >
   <property name="geometry" >
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>271</width>
     <height>711</height>
    </rect>
   </property>
   <property name="title" >
    <string>Sensors</string>
   </property>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Motor 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMotor1mA" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>0 mA</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QwtPlot" name="qwtPlotCurrent1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnClear</tabstop>
 </tabstops>
 <resources>
  <include location="../direcs-remote.qrc" />
 </resources>
 <connections/>
</ui>
